CONTRIBUTORS.yaml

```diff
@@ -14,14 +14,21 @@
 ---
 
 # two dummy entries so that autogenerated tutorials do not fail to build
-contributor1:
-    name: Contributor A (Please add contributors to the CONTRIBUTORS.yaml file)
-    halloffame: "no"
-    joined: 2020-06
-contributor2:
-    name: Contributor B (Please list all contributors of this tutorial in the metadata section)
-    halloffame: "no"
-    joined: 2020-06
+kenza12:
+    name: Kenza Bazi-Kabbaj
+    email: kenza.bazi-kabbaj@inrae.fr
+
+tduigou:
+    name: Thomas Duigou
+    email: thomas.duigou@inrae.fr
+
+breakthewall:
+    name: Joan Hérisson
+    email: joan.herisson@univ-evry.fr
+
+guillaume-gricourt:
+    name: Guillaume Gricourt
+    email: guillaume.gricourt@inrae.fr
```

Comment:
Could you please insert the entries where they belong alphabetically in the full list?

Reply:
Ok, done.